data/quarg/quarg missions.txt

```diff
@@ -274,3 +274,52 @@ mission "Quarg Ships Atrocity"
 	on offer
 		conversation "quarg imprisonment"
 
+
+
+mission "Quarg Pug Arfecta Warning"
+	minor
+	landing
+	source
+		attributes "quarg"
+		attributes "ringworld"
+	to offer
+		has "First Contact: Quarg: offered"
+		has "ship model: Pug Arfecta"
+	on offer
+		conversation
+			branch flagship
+				has "flagship model: Pug Arfecta"
+			`As soon as you land, you see a crowd of Quarg gathering around one of your escorts, the Pug Arfecta. The captain of the ship looks shaken as the Quarg surround them, seemingly being questioned by a rather tall Quarg, even by Quarg standards. The captain looks around and, seeing you, points in your direction, causing the Quarg to come your way. The tall Quarg towers over you and gives you a piercing stare.`
```

Comment:
What if this is your flagship?

Reply:
Then it goes to the label from the branch right above this line.

Reply:
Oh yeah I'm blind, phone review ig
Sorry